express_relay/sdk/js/src/index.ts

```diff
@@ -0,0 +1,285 @@
+import type { paths } from "./types";
+import createClient, { ClientOptions } from "openapi-fetch";
+import {
+  Address,
+  encodeAbiParameters,
+  encodePacked,
+  Hex,
+  isAddress,
+  isHex,
+  keccak256,
+} from "viem";
+import { privateKeyToAccount, sign, signatureToHex } from "viem/accounts";
+
+/**
+ * ERC20 token with contract address and amount
+ */
+export type TokenQty = {
+  contract: Address;
+  amount: bigint;
+};
+
+/**
+ * Bid information
+ */
+export type BidInfo = {
+  /**
+   * Bid amount in wei
+   */
+  amount: bigint;
+  /**
+   * Unix timestamp for when the bid is no longer valid in seconds
+   */
+  valid_until: bigint;
```

Comment:
are these names intentionally in `snake_case`? I think we usually use camelCase for everything in typescript